tests/e2e/set-ffi-env-e2e

```diff
@@ -37,6 +37,19 @@ source "${SCRIPT_DIR}"/lib/diskutils
 source "${SCRIPT_DIR}"/lib/repoutils
 
 # GLOBALS
+# Configuration Constants
+readonly BLUECHI_CONTROLLER_CONF="/etc/bluechi/controller.conf"
+readonly BLUECHI_AGENT_CONF="/etc/bluechi/agent.conf.d/00-default.conf"
+readonly QM_BLUECHI_AGENT_CONF="/etc/qm/bluechi/agent.conf.d/agent.conf"
+readonly QM_CONTAINER_DROPIN_DIR="/etc/containers/systemd/qm.container.d"
+readonly BLUECHI_SOCKET_DROPIN="${QM_CONTAINER_DROPIN_DIR}/bluechi-socket.conf"
+readonly BLUECHI_SOCKET_HOST="/run/bluechi/bluechi.sock"
+readonly BLUECHI_SOCKET_CONTAINER="/run/bluechi.sock"
+readonly DEFAULT_NODE_NAME="localrootfs"
+readonly DEFAULT_HOSTNAME="localrootfs"
+readonly QM_SETUP_SCRIPT="/usr/share/qm/setup"
+
+# Globals
```

Comment:
A lot of these globals aren't used, just exported here (e.g. the `CONFIG_NODE_AGENT_PATH`). For some I couldn't even find name matches in other files where these could be used. Please check if these variables can be pruned.

Reply:
@Yarboa and @dougsland, can you please take a look so I don't delete necessary globals.

Reply:
If any global would be necessary, it would break some test. If it doesn't break a test, its not necessary :D